options/locale/locale_en-US.ini

```diff
@@ -3224,6 +3225,9 @@ config.cache_adapter = Cache Adapter
 config.cache_interval = Cache Interval
 config.cache_conn = Cache Connection
 config.cache_item_ttl = Cache Item TTL
+config.cache_test = Test Cache
+config.cache_test_failed = Failed to probe the cache: %v.
+config.cache_test_succeeded = Cache test successful, got a response in %s.
```

Comment:
```suggestion
config.cache_test_succeeded = Cache test successful, response received on average in %s.
```

cc @delvh ?

Reply:
or should I remove the `/3` instead??

Reply:
I'll rm /3

Reply:
`/3`?

Reply:
@delvh see 97c0c8f58ce33e4ced374ed8253cbfa4993af1b0

My first idea was: make 3 cache func (del,set,get) and then divide by 3 to get the average time ...

but the wording now is about the whole cache test and what it's run time had ... so i removed it